pages/view_emp_list.py

```diff
@@ -31,6 +34,39 @@ def search_by_name(self, name):
         self.wait.until(expected_conditions.presence_of_element_located(
             [By.CSS_SELECTOR, f'#empsearch_employee_name_empName[value="{name}"]']))
 
+
+    def find_created_employee_by_id(self, user_id):
+        '''not optimised function, gets all IDs from all pages, returns True is user ID is found'''
+        browser = self.browser
+        current_url = browser.current_url
+        browser.find_element(*self.header_view_employee_list_btn).click()
```

Comment:
By the time this step is called the test should already be on this page, so no need to re-open the page again.